examples/complicated/validators.go

```diff
@@ -226,23 +222,19 @@ func (r User) Validate() error {
 	if err := r.AliasOnAliasWithCustomValidate.ValidateAlias(); err != nil {
 		errs.AddField("AliasOnAliasWithCustomValidate", err)
 	}
-	for k, v := range r.MapOfMap {
-		_ = k
-		_ = v
-		if len(v) < 1 {
-			errs.AddFieldf(fmt.Sprintf("MapOfMap.%v", k), "less items than 1")
-		}
-		for k, v := range v {
-			_ = k
-			_ = v
-			if utf8.RuneCountInString(v) < 3 {
-				errs.AddFieldf(fmt.Sprintf("v.%v", k), "shorter than 3 chars")
+	for MapOfMapkey8, MapOfMapvalue8 := range r.MapOfMap {
```

Comment:
Now `len()` in name looks excessive. Uniqueness is guaranteed by `baseName` and suffix.
Also could you please make `key` and `value` suffix in camelcase for readability?